TESTING.md

```diff
@@ -0,0 +1,76 @@
+
+## Manual Tests
+
+On our manual tests, we aim to do some [sanity checks](https://en.wikipedia.org/wiki/Sanity_check) to see if the main features are working as expected. Usually, we run it on each Tracetest release.
+
+### Testing Tracetest setup
+
+This is a simple test to check if Tracetest is working correctly given it was provisioned with one Tracing Data Store.
+
+The steps that we should follow are:
+
+- [ ] Open WebUI and go to `/settings` page. The provisioned Data Store should be selected.
+- [ ] Run `tracetest datastore export -d {provisioned_datastore}` and check if the data was exported correctly.
+- [ ] Create a test on WebUI that calls a demo API (like [Pokeshop](https://docs.tracetest.io/live-examples/pokeshop/overview) or [Open Telemetry Store](https://docs.tracetest.io/live-examples/opentelemetry-store/overview)). This test should fetch traces correctly and run without errors.
+
+### Checklist on version release
+
+This is the entire checklist on what we should do to assert that Tracetest is working fine on each version release. On each version release, we can copy the contents of this checklist and open a Github Discussion to start each test.
+
+- [ ] Check if our release pipeline on [Release Tracetest](https://github.com/kubeshop/tracetest/actions/workflows/release-version.yml) workflow on Github Actions worked correctly.
+- [ ] Test CLI update on MacOS via homebrew
+- [ ] Test CLI update on MacOS via curl script
+- [ ] Test CLI update on Linux via APT
+- [ ] Test CLI update on Linux via YUM
+- [ ] Test CLI update on Linux via curl script
+- [ ] Test CLI update on Windows via chocolatey
+- [ ] Test CLI update on Windows via manual download
+- [ ] Test server installation via CLI with Docker Compose and no demo API
+- [ ] Test server installation via CLI with Docker Compose and demo API
+- [ ] Test server installation via CLI with Kubernetes and no demo API
+- [ ] Test server installation via CLI with Kubernetes and demo API
+- [ ] Double check [Detailed installation](https://docs.tracetest.io/getting-started/detailed-installation) doc and see if everything is documented correctly
+
+- [ ] Test Tracetest setup with [No tracing example](https://github.com/kubeshop/tracetest/tree/main/examples/tracetest-no-tracing) by checking the last run for our CI on **Test examples** step
+- [ ] Test Tracetest setup with [Jaeger example](https://github.com/kubeshop/tracetest/tree/main/examples/tracetest-jaeger) by checking the last run for our CI on **Test examples** step
+- [ ] Test Tracetest setup with [OpenSearch example](https://github.com/kubeshop/tracetest/tree/main/examples/tracetest-opensearch) by checking the last run for our CI on **Test examples** step
+- [ ] Test Tracetest setup with [Tempo example](https://github.com/kubeshop/tracetest/tree/main/examples/tracetest-tempo) by checking the last run for our CI on **Test examples** step
```

Comment:
this can be confusing to read. if those examples don't build, PR's shouldn't be built, and the examples aren't tested on release. I'd remove these lines

Reply:
Makes sense! Changing it.